backend/routes/group.py

```
@@ -48,3 +53,81 @@
+    return result
+
+@router.get("/{university_id}/departments", response_model=List[str])
+def get_all_departments(university_id: int, db: Session = Depends(get_db)):
+    university = db.query(University).filter(University.id == university_id).first()
+    if not university:
+        raise HTTPException(status_code=404, detail="University not found")
+    return university.departments
+
+
+@router.get("/posts/university/{university_name}/department/{department_name}")
+def get_post_ids_by_department(
```

Ruff (0.8.2)

Missing return type annotation for public function `get_post_ids_by_department`

(ANN201)